meta-koi/recipes-android/android/android_koi-o.bb

```diff
@@ -0,0 +1,46 @@
+inherit gettext
+
+SUMMARY = "Downloads the Snapdragon Wear 2100/3100 /usr/libexec/hal-droid and /usr/include/android folders and installs them for libhybris"
+LICENSE = "CLOSED"
+
+SRC_URI = "https://dl.dropboxusercontent.com/s/4d8pkmd62rxl464/hybris-o-msm8909.tar.gz;name=hybris"
```

Comment:
Thanks for the reminder of 'tweak the tarball'. I've created an issue prompting for the other ports to be updated as well: https://github.com/AsteroidOS/meta-smartwatch/issues/172

Reply:
ah, makes sense. I'll see what I can do with the magic of interactive rebasing

Reply:
But as mentioned, this is for future reference only. The PR is fine, but for future PRs I would recommend to split those kind of commits to a `android-init: Remove non-existent services` and `linux: Fix Bluetooth`